Atlas/backend/atlas/urls.py

```diff
@@ -32,6 +32,7 @@
     url(r'^cultural_heritage_item/(?P<heritage_id>[0-9]+)/image/?$', views.image_media_item.as_view()),
     url(r'^cultural_heritage_item/(?P<id>\d+)/?$', views.cultural_heritage_item_view_update_delete.as_view()),
     url(r'^cultural_heritage_item/myitems', views.cultural_heritage_item_list_user_items.as_view()),
+    url(r'^comment/', views.comment.as_view()),
```

Comment:
This url must be like Cultural_heritage_item/<Number of cultural heritage item>/comments for all comments on a spesific item. And one more url like /cultural_heritage_item/<id of cultural heritage item>/<id of comment> to retrieve or update a spesific comment

Reply:
Yeah, basically you need to use cultural_heritage_item/id/comment to create a new comment for cultural heritage item with id 'id'. You will use the same as I did for item endpoint.  ```url(r'^cultural_heritage_item/(?P<heritage_id>[0-9]+)/comment/?$',``` That way you can find the corresponding cultural heritage item from the reviews.